src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,82 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String BUY_VALUE = "buy";
+    private static final String SUPPLY_VALUE = "supply";
+    private static final String RESULT_VALUE = "result";
+    private static final int NAME_INDEX = 0;
+    private static final int NUMBER_INDEX = 1;
```

Comment:
Naming is **very** important
We do not have NAME and NUMBER
We have **operation** and **amount**
![image](https://github.com/mate-academy/jv-work-with-file/assets/20326858/8b1a7161-1ff9-41e9-ad54-66fd00ac6a72)